test/runtests.jl

```diff
@@ -21,7 +21,6 @@ end
     B = A - A
     @test B isa IntervalMatrix && B == IntervalMatrix([a₋ b₋; c₋ d₋])
 
-    # can multiply, TODO: add test
```

Comment:
While I think a proper test is really missing here, we are sure that the implementation works as expected, so I agree to remove this TODO.